File: OurTournamentAPI/OurTournamentAPI/Models/Torneo.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace OurTournamentAPI.Models
{
    public class Torneo
    {
  


        private int _IDTorneo;
        private String _nombreTorneo;
        private String _contraseniaDeAdministrador;
        private String _linkParaUnirse;
        private int _TipoDeTorneo;

        public int IDTorneo { get => this._IDTorneo; set => this._IDTorneo = value; }
        public String NombreTorneo { get => _nombreTorneo; set => _nombreTorneo = value; }
        public String ContraseniaDeAdministrador { get => _contraseniaDeAdministrador; set => _contraseniaDeAdministrador = value; }
        public String LinkParaUnirse { get => _linkParaUnirse; set => _linkParaUnirse = value; }
        public int TipoDeTorneo { get => _TipoDeTorneo; set => _TipoDeTorneo = value; }

        public Torneo(int idtorneo, string nombreTorneo, string contraseniaDeAdministrador, string linkParaUnirse,int tipodetorneo)
        {
            _IDTorneo = idtorneo;
            _nombreTorneo= nombreTorneo;
            _contraseniaDeAdministrador = contraseniaDeAdministrador;
           _linkParaUnirse = linkParaUnirse;
            _TipoDeTorneo = tipodetorneo;
        }

        public Torneo()
        {

        }



    }
}

File: ProyectoOurTorunament/app/src/main/java/com/example/ourtournament/Objetos/Torneo.java
package com.example.ourtournament.Objetos;

public class Torneo
{
    public int IDTorneo;
    public String NombreTorneo;
    public String ContraseniaDeAdministrador;
    public String LinkParaUnirse;

    public Torneo(int idtorneo, String nombreTorneo, String contraseniaDeAdministrador, String linkParaUnirse)
    {
        IDTorneo = idtorneo;
        NombreTorneo= nombreTorneo;
        ContraseniaDeAdministrador = contraseniaDeAdministrador;
        LinkParaUnirse = linkParaUnirse;
    }

    public Torneo()
    {

    }

    public int getIDTorneo() {
        return IDTorneo;
    }

    public void setIDTorneo(int IDTorneo) {
        this.IDTorneo = IDTorneo;
    }

    public String getNombreTorneo() {
        return NombreTorneo;
    }

    public void setNombreTorneo(String nombreTorneo) {
        NombreTorneo = nombreTorneo;
    }

    public String getContraseniaDeAdministrador() {
        return ContraseniaDeAdministrador;
    }

    public void setContraseniaDeAdministrador(String contraseniaDeAdministrador) {
        ContraseniaDeAdministrador = contraseniaDeAdministrador;
    }

    public String getLinkParaUnirse() {
        return LinkParaUnirse;
    }

    public void setLinkParaUnirse(String linkParaUnirse) {
        LinkParaUnirse = linkParaUnirse;
    }


}